pom.xml

```diff
@@ -248,6 +248,7 @@
                                 <infinispan.expected-log>Infinispan Server.*started in</infinispan.expected-log>
                                 <spring.cloud.server.image>quay.io/quarkusqeteam/spring-cloud-config-server:3.0</spring.cloud.server.image>
                                 <bouncycastle.bctls-fips.version>1.0.12.2</bouncycastle.bctls-fips.version>
+                                <infinispan.image>quay.io/infinispan/server:14.0</infinispan.image>
```

Comment:
This image definition is duplicated, take a look line `247`
```
<infinispan.image>quay.io/infinispan/server:14.0</infinispan.image>
```

Reply:
Removed